cmd/root.go

```diff
@@ -57,6 +60,9 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&username, "username", "u", os.Getenv("SKUID_UN"), "Skuid Platform / Salesforce Username")
 	RootCmd.PersistentFlags().StringVarP(&metadataServiceProxy, "metadataServiceProxy", "", os.Getenv("METADATA_SERVICE_PROXY"), "Proxy used to reach the metadata service")
 	RootCmd.PersistentFlags().StringVarP(&dataServiceProxy, "dataServiceProxy", "", os.Getenv("DATA_SERVICE_PROXY"), "Proxy used to reach the data service")
+	RootCmd.PersistentFlags().StringVar(&variablename, "name", "", "The display name for the variable to be set")
+	RootCmd.PersistentFlags().StringVar(&variablevalue, "value", "", "The value for the variable to be set")
+	RootCmd.PersistentFlags().StringVar(&variabledataservice, "dataservice", "", "The name of the dataservice in which to set a value")
```

Comment:
The only issue with this is how to differentiate between pliny ESCs (which aren't currently usable for anything...) and cloud warden ("default" dataservice) ESCs. I'm OK with not supporting pliny ESCs with CLI until they're good for something, and just putting to cloud warden by default.

Reply:
@acofer Ah... yikes. that feels like a distinction which the user shouldn't have to think about, like we're exposing the guts of our internal implementation details to the world. Hmm. Wonder how we could make it so that "Cloud Data Service" appears to be a single thing to the world, so that no one has to know that "Data Services" are even a thing unless they are using a Private Data Service. hmm.